keystores/src/main/java/com/palantir/conjure/java/config/ssl/TrustContextFactory.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.config.ssl;
+
+import com.palantir.conjure.java.api.config.ssl.SslConfiguration;
+
+public interface TrustContextFactory {
+    TrustContext create(SslConfiguration configuration);
+}
```

Comment:
This could use some javadoc. I might suggest moving it into `ClientConfigurations` as a nested class since it's meant specifically for use with that class.

Reply:
Handled in https://github.com/palantir/conjure-java-runtime/pull/2611/commits/a543cf6fce5a665a0c1eaae42cc3f43943c49ae2